cvmfs/upload_s3.cc

```diff
@@ -157,7 +157,11 @@ bool S3Uploader::ParseSpoolerDefinition(
     timeout_sec_ = String2Uint64(parameter);
   }
   if (options_manager.GetValue("CVMFS_S3_REGION", &region_)) {
-    authz_method_ = s3fanout::kAuthzAwsV4;
+    if (region_ == "Azure") {
```

Comment:
Perhaps it is cleaner to introduce a new config variable, e.g. `CVMFS_S3_FLAVOR=[azure|aws]`.  If it is set to `azure`, we can at the same time also validate / set other config options, namely turning of DNS style bucket addresses and perhaps enforce https entpoints.

Reply:
done with simple CVMFS_S3_FLAVOR=azure; no tuning on bucket etc yet...